lib/util/problem.js

```diff
@@ -124,6 +124,9 @@ const problems = {
 
     noConflictEntity: problem(400.32, () => 'The Entity doesn\'t have any conflict'),
 
+    // { expected: "list of expected properties", actual: "list of provided properties" }
+    unexpectedProperties: problem(400.33, ({ expected, actual }) => `Expected properties: (${expected.join(', ')}). Got (${actual.join(', ')}).`),
+
```

Comment:
Is there another existing Problem we could use instead? I see this only comes up if you pass `propertyValue` AND extra stuff like `{'propertyValue': 'meow', 'cats': '   '}`. Maybe we just accept `propertyValue` and ignore the rest.

I'm also ok leaving it as is with this new Problem because there is a test for it :)